test/index.test.ts

```diff
@@ -0,0 +1,104 @@
+import connect from 'connect';
+import request from 'supertest';
+import { IncomingMessage, ServerResponse } from 'http';
+
+import clearSiteData = require('..');
+
+type Directive = 'cache' | 'cookies' | 'storage' | 'executionContexts' | '*';
+
+const WHITELIST: Directive[] = [
+  'cache',
+  'cookies',
+  'executionContexts',
+  'storage',
+  '*',
+];
+
+function app(middleware: ReturnType<typeof clearSiteData>) {
+  const result = connect();
+  result.use(middleware);
+  result.use((_req: IncomingMessage, res: ServerResponse) => {
+    res.end('Hello world!');
+  });
+  return result;
+}
+
+describe('clearSiteData', () => {
+  it('sets the header to "*" when passed no arguments', () => {
+    return request(app(clearSiteData())).get('/')
+      .expect('Clear-Site-Data', '"*"');
+  });
+
+  it('sets the header to "*" when passed an empty object', () => {
+    return request(app(clearSiteData())).get('/')
```

Comment:
It looks like this doesn't actually pass an empty object—it should be `clearSiteData({})`.

This one was my fault, as it's also an error in the version on `master`. Would you mind fixing it in this new file?